catalogs/sources/gtfs/realtime/us-california-sfmta-gtfs-rt-sa-1844.json

```diff
@@ -0,0 +1,18 @@
+{
+    "mdb_source_id": 1844,
+    "data_type": "gtfs-rt",
+    "entity_type": [
+        "sa"
+    ],
+    "provider": "San Francisco Municipal Transportation Agency (SFMTA)",
+    "static_reference": [
+        50
+    ],
+    "urls": {
+        "direct_download": "http://api.511.org/transit/servicealerts?agency=SF",
```

Comment:
should this link give the gtfs feed directly? This only works with their API

Reply:
The authentication type indicates that it only works with an API key.